.gitignore

```diff
@@ -8,3 +8,4 @@ npm-debug.log
 .jshintrc
 .vs/
 test/unit/three.*.unit.js
+build
```

Comment:
Why have you added this line?

Reply:
Sorry, it was I mistake on my part, I did not mean to push that.